Makefile

```diff
@@ -126,21 +126,24 @@ build/compiler-rt.BUILT: build/llvm.BUILT build/wasi-libc.BUILT
 	cp -R $(ROOT_DIR)/build/llvm/lib/clang $(BUILD_PREFIX)/lib/
 	touch build/compiler-rt.BUILT
 
-# Flags for libcxx.
+# Flags for libcxx and libcxxabi.
 LIBCXX_CMAKE_FLAGS = \
     -DCMAKE_C_COMPILER_WORKS=ON \
     -DCMAKE_CXX_COMPILER_WORKS=ON \
     -DCMAKE_AR=$(BUILD_PREFIX)/bin/ar \
     -DCMAKE_MODULE_PATH=$(ROOT_DIR)/cmake \
+    -DCMAKE_VERBOSE_MAKEFILE:BOOL=ON \
```

Comment:
I don't think we need to add this here.  You can do `NINJA_FLAGS=-v` if you want this I think?

Reply:
Hmm.. actually this is already set down on line 138.. I think we should remove it but I guess that is a separate issue.

Reply:
In any case this line is a duplicate of line 138 I think and so can be removed?

Reply:
Err, I missed that there was one already.